wordle-web/src/views/gameView.vue

```diff
@@ -0,0 +1,24 @@
+<template>
+    <div class="game">
+        <h1> This is the game page </h1>
+
+    <v-card 
+    class="mx-auto"
+    max-width="600"
+    max-height="400"
+    variant="outlined"
+    >
+      <v-card-title> Stirred Word </v-card-title>
+      <v-card-text>Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Maecenas porttitor congue massa. Fusce posuere, magna sed pulvinar ultricies, purus lectus malesuada libero, sit amet commodo magna eros quis urna. Nunc viverra imperdiet enim. Fusce est. Vivamus a tellus. </v-card-text>
+    
+      <v-card-actions>
+        <v-spacer />
+      <router-link to="/" custom v-slot="{ navigate }">
+        <V-btn @click="navigate" role="link" text color="primary"> Home Page </V-btn>
+      </router-link>
```

Comment:
```suggestion
      <V-btn to="/"> Home Page </V-btn>
```
Here's a more simple way of doing the same thing. Is there a reason for doing it the way you are doing it?

Reply:
Oh, that is way simpler! I'm new at this so I just didn't know there was an easier way to do that. Thanks for pointing it out!

Reply:
Glad I could help! We are all learning :)